Sources/AnimalKingdomAPICCN/GeneratedCCN/Operations/AllAnimalsQueryCCNQuery.swift

```diff
@@ -0,0 +1,63 @@
+// @generated
+// This file was automatically generated and should not be edited.
+
+import ApolloAPI
+
+public class AllAnimalsQueryCCNQuery: GraphQLQuery {
+  public let operationName: String = "AllAnimalsQueryCCN"
+  public let document: DocumentType = .notPersisted(
```

Comment:
code generation looks like it's working wonderfully for `!`/`?` 😊

Next up, list nullability syntax

Reply:
Done!

Reply:
I think I've gone and covered each of these, but let me know if you want more. 

> CCN can only be done on fields, right? You can't designate named or inline fragments with CCN?

Fields within fragments can have designators, but fragments themselves cannot, no.